pallets/investments/src/lib.rs

```diff
@@ -595,7 +572,7 @@ where
 			amount,
 		})?;
 
-		let info = T::Accountant::info(investment_id).map_err(|_| Error::<T>::UnknownInvestment)?;
+		let _ = T::Accountant::info(investment_id).map_err(|_| Error::<T>::UnknownInvestment)?;
```

Comment:
Actually, this line could be removed and saved as read access to the database, but there is a test that checks for the error `UnknownInvestment`. If removed a future `Accountant::transfer()` fails with a different error.